build_tools/docker/nvidia/Dockerfile

```diff
@@ -50,3 +50,10 @@ COPY --from=fetch-nvidia \
 RUN apt-get install "/tmp/${NVIDIA_COMMON_DEB?}" \
   "/tmp/${NVIDIA_GL_DEB?}" \
   "/tmp/${NVIDIA_COMPUTE_DEB?}"
+
+# install cuda sdk
+RUN wget https://developer.download.nvidia.com/compute/cuda/11.6.0/local_installers/cuda-repo-debian11-11-6-local_11.6.0-510.39.01-1_amd64.deb
+RUN dpkg -i cuda-repo-debian11-11-6-local_11.6.0-510.39.01-1_amd64.deb
```

Comment:
When writing scripts please prefer long options as they provide better documentation as to the intent (bundled into the suggestion below)